src/styles/helpers/__test__/getContrastText.spec.js

```diff
@@ -0,0 +1,63 @@
+import { getContrastText } from '../color';
+import { COLOR_BRAND, COLOR_SUCCESS, COLOR_WARNING } from '../../colors';
+
+const mockDarkenColorBrand = 'rgb(0, 163, 220)';
+const mockDarkenColorSuccess = 'rgb(26, 209, 163)';
+const mockColors = [
+    {
+        background: COLOR_BRAND,
+        result: 'rgba(255, 255, 255, 1)',
+        ratio: 2.328,
+    },
+    {
+        background: 'rgb(1, 182, 245)',
+        result: 'rgba(0, 0, 0, 0.87)',
+        ratio: 2.328,
+    },
+    {
+        background: 'hsl(0, 100%, 50%)',
+        result: 'rgba(0, 0, 0, 0.87)',
+        ratio: 3.992,
+    },
+    {
+        background: COLOR_SUCCESS,
+        result: 'rgba(255, 255, 255, 1)',
+        ratio: 1.569,
+    },
+    {
+        background: mockDarkenColorBrand,
+        result: 'rgba(255, 255, 255, 1)',
+        ratio: 2.884,
+    },
+    {
+        background: mockDarkenColorSuccess,
+        result: 'rgba(255, 255, 255, 1)',
+        ratio: 1.962,
+    },
+    {
+        background: COLOR_WARNING,
+        result: 'rgba(0, 0, 0, 0.87)',
+        ratio: 1.512,
+    },
+];
+
+jest.mock('../color/getContrastRatio', () => background =>
+    mockColors.filter(v => v.background === background && v.ratio),
+);
```

Comment:
this mock will return an array with the items that pass the condition passed:
```v.background === background && v.ratio```
then only one will pass an it will return an array like:
```
[{
      background: COLOR_BRAND,
        result: 'rgba(255, 255, 255, 1)',
        ratio: 2.328,
}]
```
in order to simulate the right behavior we need to make this mock to return a number like do the getContrastRatio in real cases.